.dockerignore

```diff
@@ -23,6 +23,7 @@
 **/values.dev.yaml
 LICENSE
 README.md
+src/Maestro/MaestroApplication/
```

Comment:
Why do we have this?

Reply:
This is folder is a over a GB, we don't want to copy it onto the docker image, since it's just some service fabric stuff

Reply:
We need it since we're doing `COPY ["src/", "."]`

Reply:
Wait, how is it over a GB? It's 30kB for me.

Reply:
Honestly, I have no idea why it's so big for me, but we should never need it in this docker image, so I'd just ignore that folder, just in case

Reply:
Yeah, I am just wondering what should be the rules on selectively ignoring files/folders.
For instance, we ignore `LICENSE` and `README.md` but we don't ignore [THIRD-PARTY-NOTICES.TXT](https://github.com/dotnet/arcade-services/blob/main/THIRD-PARTY-NOTICES.TXT) or [SECURITY.md](https://github.com/dotnet/arcade-services/blob/main/SECURITY.md) or [LICENSE.TXT](https://github.com/dotnet/arcade-services/blob/main/LICENSE.TXT)...

Maybe it would be better if just selectively included bits we need instead of including everything and then ignoring some?
Maybe it would be super clear what is actually part of the new service - so we'd just list all the projects.

Reply:
The License and readme were generated by default. We can probably ignore all txt, md, png, etc.. files
Regarding selectively including things, I think that'd be pretty painful, since the Maestro and Darc lib projects are quite interconnected. I guess that is not true for Maestro actors tho..

Reply:
I don't think it's such a big deal. I'd imagine we need 7-8 or projects?
DarcLib + few more.

I don't really see that as a problem? It will be then super clear what is part of the service.

Reply:
I believe there is a lot of value in being very explicit when building docker images. These things don't change often but also nullify the chance we will get bitten by something in the future.

Reply:
Added, I was expecting much more honestly